utilities/score/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM nvidia/cuda:9.0-devel-ubuntu16.04 as cuda_devel
```

Comment:
This looks like the Docker for similarity?

Reply:
updated